.github/workflows/_deploy_kafka_test.yml

```diff
@@ -88,6 +88,31 @@ jobs:
         run: | 
           az aks get-credentials --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP  --overwrite-existing
           cd main/kafka
+
+          sudo cat /root/.docker/config.json 
+          sudo mv ~/.docker/config.json /root/.docker/config.json
+          echo "docker config.json looks like the following: "
+          sudo cat /root/.docker/config.json
```

Comment:
nit: is the `cat` before moving the docker config needed?

Reply:
It is easier to debug when errors are from genpolicy